styles.css

```diff
@@ -0,0 +1,912 @@
+  height: 3.2rem;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1rem;
+}
+
+.hidden-nav li {
+  margin: 0 0.8rem;
+}
+
+.hidden-nav a {
+  color: #fff9;
+  text-decoration: none;
+}
+
+.left-hidden-nav {
+  width: 70%;
+}
+
+.right-hidden-nav {
+  width: 30%;
+  text-align: right;
+}
+
+.hidden-nav li,
+.sticky-nav li,
+.nav-breakpoint {
+  list-style: none;
+  display: inline;
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.sticky-nav {
+  background-color: rgb(253, 253, 253);
+  position: sticky;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 10;
+  top: 0;
+  height: 4.8rem;
+  border-bottom: rgb(204, 204, 204)  solid 0.1px;
+}
+
+.sticky-nav img {
+  height: auto;
+  width: 6rem;
+  margin: 0 auto;
+}
+
+.logo {
+  align-items: center;
+  background: linear-gradient(to right bottom, #d52b96, #ff4a11);
+  height: 4.8rem;
+  width: 10rem;
+}
+
+.logo-breakpoint {
+  display: none;
+}
+
+.nav-shadow {
+  box-shadow: rgb(204, 204, 204) 0 1rem 3rem;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 4.8rem;
+  z-index: -1;
+}
+
+.sticky-nav li {
+  font-size: 1.4rem;
+  margin: 0 0.8rem;
+}
+
+.sticky-nav a {
+  color: #aaa;
+  text-decoration: none;
+}
+
+.left-sticky-nav {
+  padding: 0 1rem;
+  width: 93.5%;
+}
+
+.search-icon {
+  width: 4.8rem;
+  font-size: 1.5rem;
+  font-weight: 300;
+  align-items: center;
+  justify-content: center;
+  transform: scale(-1, 1);
+  color: #aaa;
+  height: 100%;
+  border-right: solid #aaa 0.1rem;
+}
+
+.nav-breakpoint {
+  display: none;
+}
+
+/* ------------------main---------------------- */
+
+.main-img {
+  display: grid;
+  position: relative;
+  height: 50rem;
+  margin: 1rem 2rem 1rem 1rem;
+  grid-template: repeat(2, 50%) / 65% 35%;
+  grid-gap: 1rem;
+}
+
+.grid-1 {
+  background-image:
+    url(./graphics/tnw-w.png),
+    linear-gradient(transparent, black),
+    linear-gradient(to bottom right, #ff4322b2  30%, transparent 60%),
+    url(./graphics/news-1.png);
+  background-origin: content-box, padding-box, padding-box, padding-box;
+  background-size: 50% auto, cover, cover, cover;
+  background-repeat: no-repeat;
+  position: relative;
+  grid-area: 1 / 1 / 3 / 2;
+}
+
+.grid-2 {
+  background-image: linear-gradient(transparent, black), url(./graphics/news-2.png);
+  grid-area: 1 / 2 / 2 / 3;
+}
+
+.grid-3 {
+  background-image: linear-gradient(transparent, black), url(./graphics/news-3.jpeg);
+  grid-area: 2 / 2 / 2 / 3;
+}
+
+.grid-2,
+.grid-3 {
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.grid-1,
+.grid-2,
+.grid-3 {
+  justify-content: flex-end;
+  flex-direction: column;
+  padding: 2rem;
+}
+
+.grid-1 a,
+.grid-2 a,
+.grid-3 a {
+  color: #fff9;
+  text-decoration: none;
+}
+
+.main-date {
+  position: absolute;
+  top: 1rem;
+  right: 2rem;
+}
+
+/* ---------------- Latest News Styes ------------------ */
+
+.latest-news {
+  position: relative;
+  margin: 20rem auto;
+  width: 85%;
```

Comment:
- [ ] On **desktop** your sections .latest-news, .latest-funding and .tech-section are not aligned correctly, you can fix this, simply by increasing the % width of said sections
![image](https://user-images.githubusercontent.com/53256139/88866853-79cbf080-d1d1-11ea-8ce6-196e1a34fede.png)

Reply:
Hello @kitpao thanks for the review, We fixed the requested changes and the breakpoint sizes and I reached to @Bola and explain everything and I waiting for the reply.